app/(dashboard)/settings/team/teamMemberRow.tsx

```
@@ -1,16 +1,14 @@
 "use client";
 
-import { Trash } from "lucide-react";
+import { Loader, Save, SquarePen, Trash } from "lucide-react";
 import { useEffect, useState } from "react";
 import { toast } from "sonner";
 import { useSavingIndicator } from "@/components/hooks/useSavingIndicator";
```

ESLint

[error] 'useSavingIndicator' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)